### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by DanielVernall** [vector-of-bool/dds#152] [General]
Hi all,

I am a newcomer to bpt (/dds) who was drawn to it when I started researching my own attempt at a simpler c++ build system.
I can see this project was fairly active up until 2022. Is it still active, or is the developer on hiatus?

**Comment by vector-of-bool** [comment]
Hi!

Thanks for you interest. It's currently in a suspended state, but not from my own lack of interest. I'm still using this for most of my personal projects and it "just works" in all my own cases, but I've not had the time and need to do any big changes since the last release. It hasn't received much adoption, so I haven't had a lot of push/feedback from users to make any new updates or releases.

Between work at `$real_job` and `$real_life`, I don't have as much time an energy to devote to this as I would like. There are still a lot of features and changes that I would like to someday get to, but I'm not sure if/when I will have the time to devote to the project.

**Reply by DanielVernall** [reply]
Hi!

Thanks for the response. It's good to know you're still around. I understand that there's always way more projects than there is time to implement them! Knowing that the project is in a fairly stable state is good enough for me. I'll probably try it out in my next c++ project.
